src/static/riot/analytics/analytics.tag

```diff
@@ -0,0 +1,520 @@
+        <button class="ui button" onclick="{ update_chart_resolution }" id="month-units">Monthly</button>
+    </div>
+
+
+    <div class="ui top attached tabular menu">
+        <a class="item" data-tab="competitions">competitions</a>
+        <a class="item" data-tab="submissions">submissions</a>
+        <a class="active item" data-tab="users">users</a>
+    </div>
+
+        <div class="ui bottom attached tab segment" data-tab="competitions">
+
+            <h2>Competition Data</h2>
+
+            <table>
+                <thead>
+                    <tr>
+                        <th>Competitions</th>
+                        <th>Published Competitions</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <td>{competitions}</td>
+                        <td>{competitions_published}</td>
+                    </tr>
+                </tbody>
+            </table>
+
+            <div class='chart-container'>
+                <canvas id="competition_chart"></canvas>
+            </div>
+        </div>
+
+        <div class="ui bottom attached tab segment" data-tab="submissions">
+            <h2>Submission Data</h2>
+
+            <div class='chart-container'>
+                <canvas id="submission_chart"></canvas>
+            </div>
+        </div>
+
+        <div class="ui bottom attached active tab segment" data-tab="users">
+            <h2>User Data</h2>
+
+            <table>
+                <thead>
+                    <tr>
+                        <th>Users</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <td>{users_total}</td>
+                    </tr>
+                </tbody>
+            </table>
+
+            <div class='chart-container'>
+                <canvas id="user_chart"></canvas>
+            </div>
+        </div>
+
+        <button class="ui big blue button" onclick="{ download_analytics_data }">Download</button>
+    <script>
+        var self = this
+
+        /*---------------------------------------------------------------------
+         Init
+        ---------------------------------------------------------------------*/
+        self.errors = []
+
+        self.competitionsChart;
+        self.submissionsChart;
+        self.usersChart;
+        self.time_unit = 'month';
+
+        self.competitions_data;
+        self.submissions_data;
+        self.users_data;
+
+        var start_date = {
+            year: 2018,
+            month:8,
+            day: 1,
+        }
+
+        var today = new Date()
+        var end_date = {
+            year: today.getFullYear(),
+            month: today.getMonth() + 1,
+            day: today.getDate(),
+        }
+
+        self.one("mount", function () {
+            // Semantic UI
+            $('.tabular.menu .item').tab();
```

Comment:
when doing semantic ui instantiations, (like `.tab()` or `.dropdown()`) make sure we are doing them "scoped" so we don't potentially interfere w/ other tag files.
this would look like `$('.tabular.menu .item', self.root).tab()`

Reply:
done.